src/lib/tenant/configs/contracts/scroll.ts

```diff
@@ -20,7 +21,8 @@ export const scrollTenantContractConfig = ({
   isProd,
   alchemyId,
 }: Props): TenantContracts => {
-  const TOKEN = ZERO_ADDRESS;
+  const TOKEN = process.env.NEXT_PUBLIC_SCROLL_TOKEN_ADDRESS!;
```

Comment:
Is there a reason we need to pass this down via env rather than hardcoding it into the tenant config like we typically do with the other tenants?